api/app/signals/apps/reporting/management/commands/dumpcsv.py

```diff
@@ -33,10 +37,22 @@
 
 
 class Command(BaseCommand):
+    def _str2bool(self, v):
```

Comment:
Not needed at all, see comments bellow about `action="store_true"`

Reply:
check